src/EnergyPlus/DXCoils.cc

```diff
@@ -93,6 +92,7 @@
 #include <EnergyPlus/StandardRatings.hh>
 #include <EnergyPlus/UtilityRoutines.hh>
 #include <EnergyPlus/WaterManager.hh>
+#include <EnergyPlus/ZoneTempPredictorCorrector.hh>
```

Comment:
I'm a little surprised to see the DXCoil code importing stuff from up at the ZTPC level. As the ZTPC is a manager, it ends up calling the HVAC management which ultimately calls the coils.  Importing back up that level the other way just feels a bit odd.  I understand it's working, so I'm not pressing on a big change now, but I wonder if maybe these ZoneHeatBalance variables should live in DataHeatBalance or something?  I could be wrong, just thinking outloud.

Reply:
@Myoldmopar This PR ends up moving a lot of former arrays from `DataHeatBalFanSys` into structs in ZTPC. It seemed logical at the time, trying to make this data local to the ZTPC functions. But if there's a better place to put these, this PR is as good a place as any since it's already disturbing all the places that reference old `dataHeatBalFanSys` arrays anyway.